runtime/src/tasks/qthreads/tasks-qthreads.c

```diff
@@ -732,33 +731,70 @@ static inline void wrap_callbacks(chpl_task_cb_event_kind_t event_kind,
     }
 }
 
-static aligned_t chapel_wrapper(void *arg)
+// If we stored chpl_taskID_t in chpl_task_bundleData_t,
+// this struct and the following function may not be necessary.
+typedef void (*main_ptr_t)(void);
+typedef struct {
+  chpl_task_bundle_t arg;
```

Comment:
We can get away with this not being the last thing in the bundle here because chpl_main() doesn't take an argument, right?  If it did, then the need to have that argument directly follow the fixed fields in `arg` would mean that `arg` would have to be last?